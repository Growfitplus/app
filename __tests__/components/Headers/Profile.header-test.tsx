import ProfileHeader from '@/components/Headers/Profile.header';
import { render } from '@testing-library/react-native';

describe('Progress Tab', () => {
  it('should render without crashing', () => {
    const { toJSON } = render(<ProfileHeader handleProfile={jest.fn} />);

    expect(toJSON()).toMatchSnapshot();
  });
});
