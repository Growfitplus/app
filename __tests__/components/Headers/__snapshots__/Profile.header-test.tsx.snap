// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress Tab should render without crashing 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "alignItems": "center",
      "height": 40,
      "justifyContent": "center",
      "marginRight": 40,
      "width": 40,
    }
  }
>
  <RNSVGSvgView
    bbHeight={18}
    bbWidth={16}
    fill="none"
    focusable={false}
    height={18}
    style={
      [
        {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        {
          "flex": 0,
          "height": 18,
          "width": 16,
        },
      ]
    }
    width={16}
  >
    <RNSVGGroup
      fill={null}
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        clipRule={0}
        d="M10.25 4a2.25 2.25 0 1 1-4.5 0 2.25 2.25 0 0 1 4.5 0ZM12 4a4 4 0 1 1-8 0 4 4 0 0 1 8 0ZM1.75 13c0-.69.56-1.25 1.25-1.25h10c.69 0 1.25.56 1.25 1.25v2c0 .69-.56 1.25-1.25 1.25H3c-.69 0-1.25-.56-1.25-1.25v-2ZM0 13a3 3 0 0 1 3-3h10a3 3 0 0 1 3 3v2a3 3 0 0 1-3 3H3a3 3 0 0 1-3-3v-2Z"
        fill={
          {
            "payload": 4280494648,
            "type": 0,
          }
        }
        fillRule={0}
        propList={
          [
            "fill",
            "fillRule",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
